.github/workflows/ci.yaml

```diff
@@ -6,10 +6,15 @@ on:
 
 jobs:
   unit-tests:
+    strategy:
+      matrix:
+        go:
+          - '=1.18'
+          - '^1.23'
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v3
         with:
-          go-version: '>=1.18.0'
-      - run: go test
+          go-version: ${{ matrix.go }}
+      - run: find . -name go.mod -execdir go test -v ./... \;
```

Comment:
The reason for this change is because the tests were running only in `github.com/elliotchance/orderedmap`, and not `github.com/elliotchance/orderedmap/v2`.

Reply:
BTW, now with support for Matrix, I can also add support for older versions of Go only for `github.com/elliotchance/orderedmap`, if you want.